annarosepaloma/javascript30
# 15 - LocalStorage and Event Delegation

# Related Tickets
https://codeandtheory.atlassian.net/browse/JS-1

# Description

- Shows a menu with checkboxes wherein you can check/uncheck an item and add an item.

# Validation

- Please open `15 - LocalStorage/index-START.html` on a browser.

- Check or uncheck the items and add new items on the menu.

- Refresh the page and the items should still be there including their corresponding status whether checked/unchecked.

![image](https://user-images.githubusercontent.com/43773044/150912878-e9c26536-b343-4a39-895a-e16af281f48d.png)
